cmd/konnectd/serve.go

```diff
@@ -158,8 +166,12 @@ func serve(cmd *cobra.Command, args []string) error {
 
 	// Survey support.
 	var guid []byte
-	if bs.issuerIdentifierURI.Hostname() != "localhost" {
-		guid = []byte(bs.issuerIdentifierURI.String())
+	issUrl, err := url.Parse(bootstrapConfig.ISS)
```

Comment:
Mhm why does this have to be parsed here. Bootstrap should already have it as url.URL imo.